allennlp_models/generation/dataset_readers/copynet_seq2seq.py

```diff
@@ -199,18 +150,18 @@ def text_to_instance(
         """
 
         tokenized_source = self._source_tokenizer.tokenize(source_string)
-        if self._add_start_and_end_tokens:
-            tokenized_source.insert(0, Token(START_SYMBOL))
-            tokenized_source.append(Token(END_SYMBOL))
+        if not tokenized_source:
+            # If the tokenized source is empty, it will cause issues downstream.
+            raise ConfigurationError(
```

Comment:
I don't think `ConfigurationError` is the right thing here.  We should probably move away from using it anywhere except in `FromParams` and `Registrable`, preferring `ValueError` or whatever other error is appropriate.  This one looks like a `ValueError` to me.

Reply:
Good point!